app/models/saved_claim/coe_claim.rb

```diff
@@ -1,32 +1,23 @@
 # frozen_string_literal: true
 
-require 'sentry_logging'
-
 class SavedClaim::CoeClaim < SavedClaim
-  include SentryLogging
-
   FORM = '26-1880'
 
   def send_to_lgy(edipi:, icn:)
     @edipi = edipi
     @icn = icn
 
+    # If the EDIPI is blank, throw an error
     if @edipi.blank?
-      log_message_to_sentry(
-        'COE application cannot be submitted without an edipi!',
-        :error,
-        {},
-        { team: 'vfs-ebenefits' }
-      )
+      Rails.logger.error('COE application cannot be submitted without an edipi!')
+      # return back nil and break out of CoeClaim since we don't have a EDIPI
+      return nil
```

Comment:
No need to put return nil here as the Rails.logger automagically returns nil.  You can delete the return nil statement

Reply:
the return nil breaks out of the method, without we will still try to put the application

Reply:
could be an `else` instead of `return`

Reply:
Bah I didn't see that there was something underneath the return nil.

But yes, if/else would be better in this scenario.

Reply:
now that you call it out, I agree with you. I dislike pre-emptive returns.